schemas/bedrock-kms-http.js

```diff
@@ -173,11 +173,16 @@ const zcap = {
   }
 };
 
-const postRecoverBody = {
-  title: 'postRecoverBody',
+const updateKeystoreConfigBody = {
+  title: 'updateKeystoreConfigBody',
   type: 'object',
   additionalProperties: false,
-  required: ['@context', 'controller'],
+  required: [
+    // FIXME: should sequence be required here?
+    // FIXME: should context be supported or required?
```

Comment:
So I think we need sequence -- and using it would simplify how config updates work; they are just full replacements with the next sequence number.

As for `@context`, the configs are currently just JSON. I don't see a reason to change that yet.